services/drupal/config/sync/core.entity_view_display.node.event.default.yml

```diff
@@ -58,8 +59,8 @@ content:
         value: field_location
         tokenized: ''
       description:
-        value: field_description
-        tokenized: ''
+        value: '0'
+        tokenized: '[node:add-to-cal-description]'
```

Comment:
Let's ditch this particular bit of configuration since the token no longer exists.  Let's replace it with text that says something like "This value is dynamically generated in epa_core_addtocal_alter()" to make it clear to future devs what's going on here.